tests/TestCase/Http/ContentTypeNegotiationTest.php

```diff
@@ -26,6 +26,21 @@ public function testPreferredTypeNoAccept()
         $this->assertNull($content->preferredType($request));
     }
 
+    public function testPreferredTypeFirefoxHtml()
+    {
+        $content = new ContentTypeNegotiation();
+        $request = new ServerRequest([
+            'url' => '/dashboard',
+            'environment' => [
+                'HTTP_ACCEPT' => 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8',
+            ],
+        ]);
+        $this->assertEquals('text/html', $content->preferredType($request));
+        $this->assertEquals('text/html', $content->preferredType($request, ['text/html', 'application/json']));
```

Comment:
Shouldn't this be testing with html and xml to ensure html wins with xml at 0.9?

I don't know that json is helpful here.